build.sbt

```diff
@@ -36,6 +36,7 @@ val examplesTestLibs = Seq(
   "com.twitter" %% "inject-core" % twitterReleaseVersion % "test" classifier "tests",
   "com.twitter" %% "inject-modules" % twitterReleaseVersion % "test" classifier "tests",
   "com.twitter" %% "inject-server" % twitterReleaseVersion % "test" classifier "tests",
+  "org.scalatestplus" %% "mockito-1-10" % "3.1.0.0" % "test",
```

Comment:
this is needed for the "hello-world-example/test" to run - it really should be in the CI job :)